ai_chatbots/views.py

```diff
@@ -112,6 +112,45 @@ class LLMModelViewSet(ReadOnlyModelViewSet):
     ordering_fields = ["provider", "name", "litellm_id"]
 
 
+@extend_schema(
+    parameters=[
+        OpenApiParameter(
+            name="run_readable_id",
+            type=OpenApiTypes.STR,
+            location=OpenApiParameter.PATH,
+            description="run_readable_id of the course run",
+        )
+    ],
+    responses={
+        200: OpenApiResponse(description="List of problem sets"),
+        500: OpenApiResponse(description="Error retrieving problem sets"),
+    },
+)
+class GetProblemSetList(ApiView):
```

Comment:
We should probably drop the "get" from the name of the view (and list if possible) since that is implied by the rest operation (maybe ProblemSetListView or something similar)